src/core_modules/capture-core/components/WidgetTwoEventWorkspace/OverflowMenu/Modal/UnlinkAndDeleteModal.tsx

```diff
@@ -76,14 +75,14 @@ export const UnlinkAndDeleteModal = ({
                         onClick={() => setOpenModal(false)}
                         secondary
                     >
-                        {i18n.t('No, cancel')}
+                        {i18n.t('No, cancel') as string}
                     </Button>
                     <Button
                         destructive
                         onClick={() => mutation.mutate()}
                         disabled={mutation.isLoading}
                     >
-                        {i18n.t('Yes, unlink and delete linked event')}
+                        {i18n.t('Yes, unlink and delete linked event') as string}
```

Comment:
Remove the `as string`. Remember to never add `as string` after the ` i18n.t(`) function